src/controllers/users-controller.js

```diff
@@ -0,0 +1,86 @@
+const {
+  createUser,
+  getAllUsers,
+  getUserById,
+  deleteUserById,
+  UpdateUserData,
+} = require('../services/userService.js');
+
+const getUsers = (req, res) => {
+  res.send(getAllUsers());
+};
+
+const postUser = (req, res) => {
+  const { name } = req.body;
+
+  if (typeof name !== 'string') {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = createUser(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const getUser = (req, res) => {
+  const userId = +req.params.userId;
+
+  const searchedUser = getUserById(userId);
+
+  if (!searchedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
```

Comment:
You can create `middleware` to handle repeated tasks, such as checking if id is provided and valid.